src/app/loaders/express/locals.ts

```diff
@@ -64,6 +66,7 @@ const environment = ejs.render(
     var reactMigrationRespondentRolloutStorage = "<%= reactMigrationRespondentRolloutStorage%>"
     var reactMigrationAdminRollout = "<%= reactMigrationAdminRollout%>"
     var reactMigrationAngularPhaseOutDate = "<%= reactMigrationAngularPhaseOutDate%>"
+    var reactMigrationUseFetchForSubmissions = "<%= reactMigrationUseFetchForSubmissions%>"
```

Comment:
For this too, I would try to not convert to string to remove typing risks (`"false"` is truthy 😰 ) ... (although it looks like this js chunk is actually not used anywhere anyway 🤷)
```suggestion
    var reactMigrationUseFetchForSubmissions = <%= reactMigrationUseFetchForSubmissions%>
```